src/tokenizer_rules.py

```diff
@@ -1,5 +1,5 @@
 # this file is used by ply.lex
-from errors import TokenizeException
+from .errors import TokenizeException
 
 tokens = (
     'INSTRUCTION', 'DIRECTIVE', 'COMMA', 'REGISTER', 'LABEL', 'NUMBER', 'NAME', 'COMMENT'
```

Comment:
This still needs changes (we don't want to have a `DIRECTIVE` or `INSTRUCTION` tokens but instead tokens for each of the instructions).

Probably some of the tokens could also be generated from list of mnemonics (at least those with easy match).